tests/h/models/token_test.py

```diff
@@ -8,3 +8,17 @@
 class TestToken(object):
     def test_init_generates_value(self):
         assert Token().value
+
+    def test_get_by_userid_filters_by_userid(self, db_session, factories):
+        token_1 = factories.Token()
+        token_2 = factories.Token()
+
+        assert token_1 != token_2
```

Comment:
So this isn't what you're actually testing, but you're just checking that the two tokens are different because they have to be for the test below to work, right?

Is it possible that the `Token()` factory could, rarely, return two equal tokens and this test would fail?

I think you can ensure that they're always different by doing:

```python
token_1 = factories.Token(userid='user1')
token_2 = factories.Token(userid='user2')
```